docs/reference/setup/install/get-fingerprint.asciidoc

```diff
@@ -0,0 +1,28 @@
+==== Connect clients to {es}
+
+When you start {es} for the first time, it generates a hex-coded SHA-256
+fingerprint of the generated HTTPS CA DER-encoded certificate in the terminal.
+To connect your client to {es}, copy this fingerprint and pass it as a parameter.
+Any clients that connect to {es}, such as the 
+https://www.elastic.co/guide/en/elasticsearch/client/index.html[{es} Clients],
+{beats}, {ls}, and {fleet} can verify that any certificate in the chain
+matches the fingerprint.
+
+If the auto-configuration process already completed, you can still obtain the 
+fingerprint of the security certificate. Run the following command on any
+running node in the {es} cluster that you want to connect to. Modify
+`<host>:<port>` to match where your {es} node is running.
+
+[source,sh]
+----
+echo | openssl s_client -connect <host>:<port> 2>/dev/null | openssl x509 -issuer -fingerprint -sha256 | sed 's/://g'
```

Comment:
Any specific reason for getting it from the running host instead of calculating it from the `http_ca.crt` file on disk ? I can think of 2 I guess: 

1. This version here will work even when users have replaced the auto-generated certificate with their own, but I would expect that they then could also use the actual CA certificate in the client config.
2. The client admin might not have file access to the ES node to calculate the fingerprint of the certificate on disk

_but on the other hand_

1. This is inherently less secure. There is nothing validating the certificate in the original openssl s_client call, so if a MITM is already positioned in front of the ES node, then the client would be configured to trust a malicious CA fingerprint. We lose the whole purpose of initial trust bootstrap that we're trying to achieve with the fingerprint option. 
2. Less important,  `openssl x509 -fingerprint -sha256 -in <path/to/http_ca.crt>` reads a little less scary than what you have here :) .

Reply:
@jkakavas, I modified the docs in 0a40a3b3b75832b80d7a7d48edca2e2c1c5df637 to indicate using the `http_ca.crt` file on disk, and updated the `openssl` command to match -- thanks for the less-scary syntax!